backend/chalice/api_server/requirements.txt

```diff
@@ -1,4 +1,5 @@
-PyMySQL==0.9.3
+# psycopg2<2.8 simplifies build:https://github.com/psycopg/psycopg2/issues/674
```

Comment:
Do we need to switch to this at some point or can we delete this since it's commented out for now?

Reply:
I don't anticipate a reason to upgrade in the future, but included the comment as a reminder to not upgrade beyond the `2.7.7` pin.

Reply:
Do you still prefer to remove?

Reply:
Ohh I didn't understand that that was the intent. Yeah, totally fine to leave in there, but maybe just explain it as you just did? `# Don't upgrade beyond 2.7.7 because...`

Reply:
Making clearer, thanks!